app/components/rule-row/component.js

```diff
@@ -22,6 +25,10 @@ export default Ember.Component.extend({
   actions: {
     delete() {
       this.sendAction('delete', this.get('rule'));
+    },
+    toggleTerm() {
+      $(`[data-rule="${this.get('rule.id')}"]`).toggle();
+      $(`[data-rule="${this.get('rule.id')}"]`).is(":visible") ? this.set('toggleText', collapseText) : this.set('toggleText', expandText);
```

Comment:
We should scope these jQuery selectors to this component/view, otherwise the scope is global and can lead to unexpected behaviours.  `this.$()` is the component's root element, but to select a descendant, you want:

```
this.$(`[data-rule="${this.get('rule.id')}"]`)
```